pom.xml

```diff
@@ -104,6 +104,20 @@
         <artifactId>maven-enforcer-plugin</artifactId>
         <version>3.2.1</version>
         <executions>
+          <execution>
+            <id>enforce-maven-version</id>
```

Comment:
I wonder if this can be merged with the existing `<execution>`? The example on the [usage page](https://maven.apache.org/enforcer/maven-enforcer-plugin/usage.html) makes it look as if that should be possible.